package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "private": true,
+  "scripts": {
+    "start": "webpack serve --open --config webpack.dev.js",
+    "build": "webpack --config webpack.prod.js",
+    "lint": "npx eslint \"**/*.tsx\" \"**/*.ts\" && npx stylelint \"**/*.css\" \"**/*.scss\"",
+    "lint-fix": "npx eslint --fix \"**/*.tsx\" \"**/*.ts\" && npx stylelint --fix \"**/*.css\" \"**/*.scss\""
```

Comment:
I don't think you'll need to use `npx` here since the command should already run with `node_modules`, but I may be wrong!

Reply:
yup!